modules/mono/.editorconfig

```diff
@@ -0,0 +1,14 @@
+[*.sln]
+indent_style = tab
+
+[*.{csproj,props,targets,nuspec,resx}]
+indent_style = space
+indent_size = 2
+
+[*.cs]
+indent_style = space
+indent_size = 4
+insert_final_newline = true
+trim_trailing_whitespace = true
+max_line_length = 120
+csharp_indent_case_contents_when_block = false
```

Comment:
Can you add these too?:

```ini
charset = utf-8
end_of_line = lf
```

Reply:
We already enforce this for all files by default with the root `.editorconfig` in the root of the Godot repository.